### ISSUE TITLE
multi link at once.

### ISSUE THREAD

**Issue by Sombian** [jarbun/downline#13]
its kinda hard to manually add over 3k items each thus why not support this?
such format as

```
link one
link two
so on
```

this!
you could split the string by newline or maybe even spaces.
hope this added soon.

**Comment by balkarjun** [comment]
That's a great suggestion. I've already added it to v2.0 that I'm working on [here](https://github.com/jarbun/downline/tree/v2). You can add multiple links, one per line. Here's what it looks like

![preview](https://user-images.githubusercontent.com/23068820/61586481-c2020d00-ab92-11e9-87c1-cb2d69a5f22d.png)

**Comment by balkarjun** [comment]
I'm also adding support for adding links from a text file, by either dragging the file to the input box, or browsing for the file

**Comment by Sombian** [comment]
> I'm also adding support for adding links from a text file, by either dragging the file to the input box, or browsing for the file

awesome!
though i see quite some parts that can be improved.
how about add single instance lock?
```js
if (!app.requestSingleInstanceLock()) {
  app.quit();
}

app.on("second-instance", () => {
  if (window) {
    if (window.isMinimized()) {
      window.restore();
    } window.focus();
  }
})
```
add this to background.js.
i also dev some electron stuff and yours are super light and looks clean too! well done.

**Comment by balkarjun** [comment]
Feature to add multiple links at once is available in v2.0.0-alpha.1.
The second suggestion to add a single instance lock, will be done before the stable release
